### PR TITLE
Minor content changes. Improvement to the readability of the learning lab

### PR DESCRIPTION
**Pull request by TheDarkLord247**
Don't want to merge just yet. Have some more commits on the way!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function App() {
             <p>
               While fully-connected neural networks look at data as a whole,
               CNNs look at data <em>one feature at a time</em>. </p>
-            <p>Before we move on, check out <a className="link-to-slides" href="https://teachla.uclaacm.com/classes/ml/10-cnn" target="_blank" rel="noopener noreferrer">these slides</a> to learn about how filters work.</p>
+            <p>Before we move on, check out <a className="link-to-slides" href="https://teachla.uclaacm.com/classes/ml/10-cnn" target="_blank" rel="noopener noreferrer" rel="noopener noreferrer">these slides</a> to learn about how filters work.</p>
```
Comment (mattxwang) [inline src/App.js:118]: Haha seeing your commit message, just to help out - you've duplicated the `rel` twice, I think if you remove one of them it should pass CI?

You can perform the local check with 

```js
$ npm run build
```

Comment (TheDarkLord247) [reply src/App.js:118]: 🐐

File: src/App.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ import depth from './img/depth.png';
 import FilterSizes from './components/FilterSizes.js';
 import visualizer from './img/cnn-vis.png';
 import filterGIF from './img/filter.gif';
-import greyscaleGIF from './img/greyscale.gif';
+//import greyscaleGIF from './img/greyscale.gif';
```
Comment (mattxwang) [inline src/App.js:14]: Let's remove this comment before merging!

File: src/App.js [hunk 3]
```diff
@@ -157,23 +157,24 @@ function App() {
       <section className="grey section">
         <div className="card">
           <h2 style={{textAlign:"center"}}>Look at the animation below to learn how a filter (shown as the red matrix) works on images to calculate each element of the feature map</h2>
+          <p style={{textAlign:"center"}}>(You can also explore our slides on filters (slide 28) from our <a target="_blank" href="https://docs.google.com/presentation/d/1je-mnp1E6y-wj3q_5GNHG8jJm9ckLTyoFNjF3Qj_6LM/edit?usp=sharing" rel="noopener noreferrer">CNN high-school lesson)</a></p> 
+          <p style={{textAlign:"center"}}>Note that whenever the filter hovers over a portion of the image, the corresponding element on the
```
Comment (mattxwang) [inline src/App.js:161]: If everything in this `<div>` is just `text-align: center;`, you can apply it to the parent to reduce duplication! May also be a good choice to create a `.text-center` class.

**Review by mattxwang — CHANGES_REQUESTED** [review]
Overall looks good to me, I left two nitpicky comments but I think it's good to merge. Keep on being awesome 🐐

File: src/App.js (hunk 3 above)
Comment (TheDarkLord247) [reply src/App.js:161]: will push these changes in another PR

**Review by mattxwang — APPROVED** [review]

**Pull request merged**